docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+                   volume.beta.kubernetes.io/storage-provisioner: csi.vsphere.vmware.com
+    Finalizers:    [kubernetes.io/pvc-protection]
+    Capacity:      5Gi
+    Access Modes:  RWO
+    VolumeMode:    Filesystem
+    Mounted By:    <none>
+    Events:
+      Type    Reason                 Age                From                                                                                                 Message
+      ----    ------                 ----               ----                                                                                                 -------
+      Normal  Provisioning           50s                csi.vsphere.vmware.com_vsphere-csi-controller-7777666589-jpnqh_798e6967-2ce1-486f-9c21-43d9dea709ae  External provisioner is provisioning volume for claim "default/example-vanilla-block-pvc"
+      Normal  ExternalProvisioning   20s (x3 over 50s)  persistentvolume-controller                                                                          waiting for a volume to be created, either by external provisioner "csi.vsphere.vmware.com" or manually created by system administrator
+      Normal  ProvisioningSucceeded  8s                 csi.vsphere.vmware.com_vsphere-csi-controller-7777666589-jpnqh_798e6967-2ce1-486f-9c21-43d9dea709ae  Successfully provisioned volume pvc-7ed39d8e-7896-11ea-a119-005056983fec
+```
+
+* Verify a PersistentVolume was created
+
+  Next, let's check if a `PersistentVolume` was successfully created for the `PersistentVolumeClaim` we defined above.
+
+  If it has worked you should have a `PersistentVolume` show up in the output and you should see that the `VolumeHandle`
+  key is populated, as is the case below
+
+  The `Status` should say `Bound`. You can also see the `Claim` is set to the above `PersistentVolumeClaim` name `example-vanilla-block-pvc`
+
+```text
+  $ kubectl describe pv pvc-7ed39d8e-7896-11ea-a119-005056983fec
+    Name:            pvc-7ed39d8e-7896-11ea-a119-005056983fec
+    Labels:          <none>
+    Annotations:     pv.kubernetes.io/provisioned-by: csi.vsphere.vmware.com
+    Finalizers:      [kubernetes.io/pv-protection]
+    StorageClass:    example-vanilla-block-sc
+    Status:          Bound
+    Claim:           default/example-vanilla-block-pvc
+    Reclaim Policy:  Delete
+    Access Modes:    RWO
+    VolumeMode:      Filesystem
+    Capacity:        5Gi
+    Node Affinity:   <none>
+    Message:
+    Source:
+        Type:              CSI (a Container Storage Interface (CSI) volume source)
+        Driver:            csi.vsphere.vmware.com
+        VolumeHandle:      e4073a6d-642e-4dff-8f4a-b4e3a47c4bbd
+        ReadOnly:          false
+        VolumeAttributes:      storage.kubernetes.io/csiProvisionerIdentity=1586239648866-8081-csi.vsphere.vmware.com
+                               type=vSphere CNS Block Volume
+    Events:                <none>
+```
+
+### Steps to dynamically provision a block volume in Project Pacific
```

Comment:
What about Tanzu?

Reply:
Isn't it the the same as Project Pacific?

Reply:
We talked about 3 flavors at the end of README: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/docs/book/README.md
Project Pacific refers to supervisor cluster.
Tanzu Kubernetes Grid refers to guest cluster.